dutch/bos/ontvoering/AllesKomtGoed/alleskomtgoed.md

```diff
@@ -0,0 +1 @@
+Je wordt de volgende dag met een kater wakker in [de hei](../../welkom.md).
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.